resources/schemas.json

```diff
@@ -754,47 +767,42 @@
                         "type": "string",
                         "description": "The item's value"
                     },
-                    "addedOn": {
-                        "type": "string",
-                        "format": "date-time",
-                        "description": "The date-time of addition (chain time)"
-                    },
                     "submitter": {
                         "description": "The address of the submitter",
                         "$ref": "#/components/schemas/SupportedAccountId"
                     },
                     "fees": {
```

Comment:
fees are common to all items now, maybe move that to ItemLifecycleView (and maybe rename as fees are not strictly speaking part of lifecycle).